cabal.project

```diff
@@ -282,7 +282,7 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-config
-  tag: e9de7a2cf70796f6ff26eac9f9540184ded0e4e6
+  tag: fe7855e981072d392513f9cf3994e0b6eba40d7d
   --sha256: 1wm1c99r5zvz22pdl8nhkp13falvqmj8dgkm8fxskwa9ydqz01ld
```

Comment:
```suggestion
  --sha256: 1q9vscnx0i2xfajymzssj9id2w3qs5ssjlrl00f78jqi0zwmgq7z
```

Reply:
How did you get that hash so quick?

Reply:
```shell
$ nix-prefetch-git https://github.com/input-output-hk/cardano-config fe7855e981072d392513f9cf3994e0b6eba40d7d
```

Reply:
Thanks!

Reply:
I ended up doing this:

```
nix-shell -p nix-prefetch-git --command 'nix-prefetch-git https://github.com/input-output-hk/cardano-config fe7855e981072d392513f9cf3994e0b6eba40d7d'
```

Reply:
Right, I've got that in my environment as I ran
```
nix-env --install --attr nix-prefetch-git
```

Reply:
Nice.  Thanks!

```
$ nix search nix-prefetch-git | cat
warning: using cached results; pass '-u' to update the cache
* nixpkgs.nix-prefetch-git (nix-prefetch-git)
  Script used to obtain source hashes for fetchgit

* nixpkgs.nix-prefetch-github (nix-prefetch-github)
  Prefetch sources from github

* nixpkgs.python38Packages.nix-prefetch-github (python3.8-nix-prefetch-github)
  Prefetch sources from github

* nixpkgs.python39Packages.nix-prefetch-github (python3.9-nix-prefetch-github)
  Prefetch sources from github

$ nix-env -i --attr nixpkgs.nix-prefetch-git
replacing old 'nix-prefetch-git'
installing 'nix-prefetch-git'
```